### ISSUE TITLE
404 on Assets

### ISSUE THREAD

**Issue by brandonparsons** [akira/exq_ui#49]
Hi there,

I had ExqUI working before, but now I'm getting 404's when it tries to load the assets (`/assets/vendor.css`, `/assets/exqui.css`, `/assets/vendor.js`, `/assets/exqui.js`). The HTML page is being served correctly (I can go to `/exq` in my app), it's just failing to load the assets.

Any idea what might be causing this?

I have HTTP basic auth set up on accessing `/exq`, but this was working previously.

This is part of a Phoenix app. I have the following in `config.exs`:

```elixir
config :exq_ui, server: false
```

And this in my router:

```elixir
  pipeline :exq do
    plug :accepts, ["html"]
    plug :fetch_session
    plug :fetch_flash
    # plug :protect_from_forgery ## Docs show pipeline w/o CSRF, Exq UI fails if include this
    plug :put_secure_browser_headers
    plug ExqUi.RouterPlug, namespace: "exq"
  end

  scope "/exq", ExqUi do
    pipe_through [:http_basic_auth, :exq]
    forward "/", RouterPlug.Router, :index
  end
```

**Comment by akira** [comment]
@j-mcnally I noticed we've added [rootUrl](https://github.com/akira/exq_ui/commit/adca0c9ea60d6df749600bf0d563a4fd99d2d9ba#diff-aed02745042475be3ff6145b3a8331fcR19) when upgrading Ember. Should that be populated to reflect root context (as it could be served from something like `/exq/`  or the root context `/`).

**Comment by brandonparsons** [comment]
For what it's worth - it seemed like no matter what config options I set, it was always requesting assets from `/assets/<file>.<ext>`.

It does seem to make sense that if I've got `exq_ui` set up to serve from `/exq`, those asset requests should go through the same path - `/exq/assets/<file>.<ext>`.

**Comment by akira** [comment]
@j-mcnally I removed rootUrl from the html to unblock this issue, let me know if there is a better approach. 

@brandonparsons I published 0.8.6, which works now.  Let me know if this issue is fixed.

**Comment by brandonparsons** [comment]
I upgraded to 0.8.6, and this issue appears to be fixed. Thanks for the quick turnaround!
